russian/speak_to_neighbor/speak_to_neighbor.md

```diff
@@ -1,2 +1,7 @@
 Я вышел на лестничную площадку и позвонил соседям.
 Если ни ни кого из соседей на площадке не оказалось дома, то я [поднимусь этажом выше](upstairs/upstairs.md)
+
+Если же зубака дома, то я вручу ей:
+[ведро бобров](vedro/vedro.md)
+[бобро ведров](bobro/bobro.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.